scripting/pugsetup.sp

```diff
@@ -2237,3 +2244,30 @@ stock bool PermissionFromString(const char[] permissionString, Permission& p,
 
   return true;
 }
+
+public int GetTotalPlayers()
+{
+    int count;
+    LoopClients(client)
+    {
+        count++;
+    }
+    
+    return count;
+}
+
+public void OnClientDisconnect(int client)
+{
+    if (g_GameState == GameState_Live)
+    {    
+        if(GetTotalPlayers() <= 2)
+        {
+            GetConVarBool(g_EndGameOnLastPlayerInServerCvar);
```

Comment:
Use if Statement here instead of this, I dont know if it works or not but even if it works it will be no longer supported in future

Reply:
done.